homeassistant/components/onewire/sensor.py

```diff
@@ -92,15 +103,17 @@ def hb_info_from_type(dev_type="std"):
 
 def setup_platform(hass, config, add_entities, discovery_info=None):
     """Set up the one wire Sensors."""
-    base_dir = config[CONF_MOUNT_DIR]
-    owport = config[CONF_PORT]
+    base_dir = config.get(CONF_MOUNT_DIR)
+    owport = config.get(CONF_PORT)
     owhost = config.get(CONF_HOST)
+    _LOGGER.info("onewire initializing using : %s %s", owhost, owport)
```

Comment:
Debug max.

Reply:
We start logging messages with capital letter. I suggest starting with the verb.